src/main/java/com/google/sps/servlets/LocationServlet.java

```diff
@@ -0,0 +1,106 @@
+package com.google.sps.servlets;
```

Comment:
I didn't see anything about the locationservlet in the description for this CL? Any reason why it being added here?

Reply:
That's odd, not sure why it's here.